pkg/settlement/swap/swap.go

```diff
@@ -212,3 +223,73 @@ func (s *Service) Handshake(peer swarm.Address, beneficiary common.Address) erro
 	}
 	return nil
 }
+
+// LastSentCheque returns the last sent cheque for the peer
+func (s *Service) LastSentCheque(peer swarm.Address) (*chequebook.SignedCheque, error) {
+
+	common, known, err := s.addressbook.Beneficiary(peer)
+
+	if err != nil {
+		return nil, errors.New("error translating swarm address to chequebook address")
```

Comment:
Word error is redundant in the error message and the original error is not propagated.

Reply:
changed to return nil, err